commands/intensify.js

```diff
@@ -0,0 +1,74 @@
+const Command = require('./command.js')
+const Image = require('../models/image.js');
+const ImageLoader = require('../util/image-loader.js')
+const ImageUploader = require('../util/image-uploader.js')
+const UtilArray = require('../util/array.js')
+const {TRANSPARENT_BLACK} = require('../constants.js')
+
+class Intensify extends Command {
+    static help() {
+        return `*Usage:* \`emojitools intensify <url>\``
+    }
+
+    async render() {
+        const image = await ImageLoader.fromUrl(this.getUrl());
+        const frame_delay = 5;
+
+        const frames = image['_frames'];
+
+        let expanded_and_split_frames = UtilArray.flatmap(frames.map((frame, idx) => {
+            const expandedFrame = this.expandFrame(frame);
+            return this.splitFrame(expandedFrame, frame_delay);
```

Comment:
you should be able to do this in the `effects` command, without needing to create a separate command just for intensifying. `Frame.transformFrames` allows you to return an array of frames from the transformation callback